pkg/encryption/utils.go

```diff
@@ -0,0 +1,88 @@
+package encryption
```

Comment:
Is this now duplicated with the cookies package?

Reply:
This is definitely related to this PR cleaning that organization up: https://github.com/oauth2-proxy/oauth2-proxy/pull/538

Reply:
Right I see, this content has just been moved from `cipher.go` to this file, makes sense now